infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -223,14 +223,6 @@ private void fillHole(CorfuPayloadMsg<FillHoleRequest> msg, ChannelHandlerContex
         }
     }
 
-    @ServerHandler(type = CorfuMsgType.TRIM)
-    private void trim(CorfuPayloadMsg<TrimRequest> msg, ChannelHandlerContext ctx, IServerRouter r) {
-        TrimRequest req = msg.getPayload();
-        batchWriter.trim(req.getAddress().getSequence(), req.getAddress().getEpoch());
-        //TODO(Maithem): should we return an error if the write fails
-        r.sendResponse(ctx, msg, CorfuMsgType.ACK.msg());
-    }
-
     @ServerHandler(type = CorfuMsgType.PREFIX_TRIM)
     private void prefixTrim(CorfuPayloadMsg<TrimRequest> msg, ChannelHandlerContext ctx,
```

Comment:
Shouldn't we remove the trim logic from the LogUnitClient as well?
And also clean up, by removing the trim CorfuMsgType.

Reply:
Done.